op-acceptor/nat.go

```diff
@@ -260,9 +303,23 @@ func (n *nat) printResultsTable(runID string) {
 			if i == len(gate.Tests)-1 && len(gate.Suites) == 0 {
 				prefix = "└──"
 			}
+
+			// Get a display name for the test
+			displayName := testName
+			if displayName == "" && test.Metadata.Package != "" {
+				// For package names, use a shortened version to avoid wrapping
+				pkgParts := strings.Split(test.Metadata.Package, "/")
+				if len(pkgParts) > 0 {
+					displayName = pkgParts[len(pkgParts)-1] + " (package)"
+				} else {
+					displayName = test.Metadata.Package
+				}
+			}
+
+			// Display the test result
```

Comment:
Small nit: these two blocks are identical to the ones above, I'd maybe just DRY into some private function

Reply:
Good call; I've now fixed all these up into one function. Cheers!